src/compiler/mir/typetable.rs

```diff
@@ -383,7 +383,7 @@ pub struct Field {
 pub struct FieldId(u32);
 
 impl FieldId {
-    fn new(id: u32) -> FieldId {
+    pub(super) fn new(id: u32) -> FieldId {
```

Comment:
Instead of making `new` accessible where it should not be, perhaps I can add a function to convert `FieldId` to a u32?  Since all I need this for is testing.

Reply:
I might be able to add a compiler config so it only compiles for testing too.